tool/jt.rb

```diff
@@ -1693,7 +1693,7 @@ def install_jvmci
     dir = File.expand_path("..", TRUFFLERUBY_DIR)
     java_home = chdir(dir) do
       if LINUX
-        jvmci_version = "jvmci-0.36"
+        jvmci_version = "jvmci-0.38"
```

Comment:
Can we keep this in sync with CI somehow?

Reply:
I was thinking to read it from `ci.jsonnet` or an ENV var.
The problem is we don't necessarily have the same JVMCI version in the release and our repo, in the commit with the release tag.

Reply:
Now it defaults to the JVMCI version in `ci.jsonnet`.